json/src/test/java/feign/json/JsonEncoderTest.java

```diff
@@ -43,13 +44,13 @@ public void setUp() {
   @Test
   public void encodesArray() {
     new JsonEncoder().encode(jsonArray, JSONArray.class, requestTemplate);
-    assertEquals("[{\"a\":\"b\",\"c\":1},123]", new String(requestTemplate.body(), UTF_8));
+    JSONAssert.assertEquals("[{\"a\":\"b\",\"c\":1},123]", new String(requestTemplate.body(), UTF_8), false);
   }
 
   @Test
   public void encodesObject() {
     new JsonEncoder().encode(jsonObject, JSONObject.class, requestTemplate);
-    assertEquals("{\"a\":\"b\",\"c\":1}", new String(requestTemplate.body(), UTF_8));
+    JSONAssert.assertEquals("{\"a\":\"b\",\"c\":1}", new String(requestTemplate.body(), UTF_8),false);
```

Comment:
probably going to fail due to code format....

build it locally, and maven will fix this for you

Reply:
Sorry for the format issue. I think I've fixed that now.